virt/virt-web-console-overview.adoc

```diff
@@ -623,7 +628,214 @@ You can create, edit, and clone VirtualMachine templates on the Templates page.
+|====
+=====
+
+[id="ui-template-details-details_{context}"]
+==== Details tab
+
+You can configure a custom template on the *Details* tab.
+
+.Details tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|Name |Template name.
+|Namespace |Template namespace.
+|Labels |Click the edit icon to edit the labels.
+|Annotations |Click the edit icon to edit the annotations.
+|Display name |Click the edit icon to edit the display name.
+|Description |Click the edit icon to enter a description.
+|Operating system |Operating system name.
+|CPU\|Memory |Click the edit icon to edit the CPU\|Memory request.
+
+The number of CPUs is calculated by using the following formula: `sockets * threads * cores`.
+|Machine type |Template machine type.
+|Boot mode |Click the edit icon to edit the boot mode.
+|Base template |Name of the base template used to create this template.
+|Created at |Template creation date.
+|Owner |Template owner.
+|Boot order |Template boot order.
+|Boot source |Boot source availability.
+|Provider |Template provider.
+|Support |Template support level.
+|GPU devices |Click the edit icon to add a GPU device.
+|Host devices |Click the edit icon to add a host device.
+|====
+=====
+
+[id="ui-template-details-yaml_{context}"]
+==== YAML tab
+
+You can configure a custom template by editing the YAML file on the *YAML* tab.
+
+.YAML tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|*Save* button |Save changes to the YAML file.
+|*Reload* button |Discard your changes and reload the YAML file.
+|*Cancel* button |Exit the *YAML* tab.
+|*Download* button |Download the YAML file to your local machine.
+|====
+=====
+
+
+[id="ui-template-details-scheduling_{context}"]
+==== Scheduling tab
+
+You can configure scheduling on the *Scheduling* tab.
+
+.Scheduling tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+
+|Node selector
+|Click the edit icon to add a label to specify qualifying nodes.
+
+|Tolerations
+|Click the edit icon to add a toleration to specify qualifying nodes.
+
+|Affinity rules
+|Click the edit icon to add an affinity rule.
+
+|*Descheduler* switch
+|Enable or disable the descheduler. The descheduler evicts a running pod so that the pod can be rescheduled onto a more suitable node.
+
+|Dedicated resources
+|Click the edit icon to select *Schedule this workload with dedicated resources (guaranteed policy)*.
+
+|Eviction strategy
+|Click the edit icon to select *LiveMigrate* as the VirtualMachineInstance eviction strategy.
+|====
+=====
+
+[id="ui-template-details-network-interfaces_{context}"]
+==== Network interfaces tab
```

Comment:
missing bold. It could be that these titles don't need it; I couldn't find the specific guidance so I wanted to note it. Here is what I found: https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements